src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -324,11 +324,9 @@ Json::Value Profile::ToJson() const
     MTSM_PROFILE_SETTINGS(PROFILE_SETTINGS_TO_JSON)
 #undef PROFILE_SETTINGS_TO_JSON
 
-    // Font settings
-    const auto fontInfoImpl = winrt::get_self<FontConfig>(_FontInfo);
-    if (fontInfoImpl->HasAnyOptionSet())
+    if (auto fontJSON = winrt::get_self<FontConfig>(_FontInfo)->ToJson(); !fontJSON.empty())

```

Comment:
interesting. serialize first, then check!

Reply:
Yep! I felt like that it made the code a lot simpler.